games-strategy/fheroes2/fheroes2-1.0.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A recreation of Heroes of Might and Magic II game engine"
+HOMEPAGE="https://ihhub.github.io/fheroes2/"
+SRC_URI="https://github.com/ihhub/fheroes2/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+image"
+
+BDEPEND="sys-devel/gettext"
+DEPEND="
+	image? ( media-libs/sdl2-image[png] )
+	media-libs/libsdl2
+	media-libs/sdl2-mixer
+	sys-libs/zlib:=
+"
+RDEPEND="${DEPEND}
+	virtual/libintl
+"
+
+src_prepare() {
+	default
+	sed -i -e "s:doc/README.txt:doc/INSTALL.md:g" -e "s:LICENSE::g" \
+		CMakeLists.txt || die
+	cmake_src_prepare
+}
+
+src_configure() {
+	mycmakeargs=(
+		-DUSE_SYSTEM_LIBSMACKER=OFF # broken
```

Comment:
Is there an upstream issue that can be linked to track this?

Reply:
Buildsystem tries to search dev-games/libsmacker via pkg-config, but system package don't provides .pc-files. Actually they uses customized bundled libsmacker, and they stated that vanilla libsmacker broken to play actual data.